source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -15,15 +24,36 @@ Controls Glossary
+
+   counter-electromotive force
+      A :term:`voltage` generated in a spinning motor. The voltage is a result of the fact that has a coil of wire rotating near a magnet. See `Counter-electromotive_force <https://en.wikipedia.org/wiki/Counter-electromotive_force>`__ on wikipedia for more info.
+
+   current
+      The flow of electrons through a conductor. Current is described with a unit of "Amps" (or simply "A"), and is measured at a single point in a circuit. One amp is equal to :math:`6241509074000000000` electrons moving past the measurement point in one second.
+
    dynamics
       A branch of physics concerned with the motion of bodies under the action of forces. In modern control, systems evolve according to their dynamics.
 
+   derivative
+      A mathematical operation which evaluates the "rate-of-change" of a function at a given point. See `derivative <https://en.wikipedia.org/wiki/Derivative>`__ on wikipedia for more info.
+
    error
       :term:`Reference <reference>` minus an :term:`output` or :term:`state`.
 
+   exponential search
+      An iterative process of finding a specific value within a wide search range by applying a multiplicative factor to the search value. See `exponential search <https://en.wikipedia.org/wiki/Exponential_search>`__ on wikipedia for more info.
+
+   exponential smoothing
+      The usage of exponential window function in a convolution with an input signal. It is a very common way to implement a simple low-pass filter, because the convolution operation simplifies down to a very simple set of math operations on the current input and previous output. See `exponential smoothing <https://en.wikipedia.org/wiki/Exponential_smoothing>`__ on wikipedia for more info.
+
    gain
       A proportional (scalar) value that relates the magnitude of an input signal to the magnitude of an output signal. In the signal-dimensional case, gain can be thought of as the proportional term of a PID controller. A gain greater than one would amplify an input signal, while a gain less than one would dampen an input signal. A negative gain would negate the input signal.
 
+   Gaussian distribution
+      A special mathematical function that describes distributions of average values. As more and more data goes into an average, its distribution will almost always get closer to a Gaussian distribution, even if the individual components of the average do not themselves follow a Gaussian distribution. The graph of a Gaussian function is a "bell curve" shape. This function is described by its mean (the location of the "peak" of the bell curve) and variance (a measure of how "spread out" the bell curve is). See `Gaussian distribution <https://en.wikipedia.org/wiki/Gaussian_function>`__  on wikipedia for more info.
```

Comment:
Should probably also work in the term normal into this description. I'm not sure the use of the term average is very clear. Is it saying that as more samples of a random variable are taken, the more it looks gaussian. Or is it saying that the combination of many random variables is gaussian, even when the individual distributions are not.

Reply:
It should say the latter. https://en.wikipedia.org/wiki/Central_limit_theorem

Reply:
In reviewing this, I've got another idea actually.... less is more.  I'm not opposed to having that info _somewhere_. However, keep in mind the goal of these glossary entries is to provide _just barely enough_ info for an FRC student to understand the term in an FRC context (and the wikipedia link is the jumping off point for "if you want to know more".

The only usage of the term currently is in the description of a Kalman filter. I'd argue that the central limit theorem explains _why_ the "model states as gaussians" assumption is valid, but isn't critical to just understanding the fact that it is. From that perspective, trimming the definition down to just saying what it is (function that describes distribution of numbers impacted by randomness), and leaving the rest to wikipedia is what I'll push for next review.